packages/database/src/migrations/20210126013657-AddK13C580YPositiveMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const MAIN_OVERLAY_ID = 'PG_STRIVE_K13_C580Y_Positive';
+const LINKED_MEASURE_OVERLAY_ID = 'PG_STRIVE_K13_C580Y_Positive_Count';
+const NEW_OVERLAY_GROUP_ID = generateId();
```

Comment:
My understanding is that it is not common to execute much code at the file level, instead doing everything except variable declaration in functions. This is (maybe, partly?) because the code will run when the module is first imported (see https://javascript.info/modules-intro#a-module-code-is-evaluated-only-the-first-time-when-imported) rather than when the `up` function is called. This means, for instance, that even if up were never called, we would still have to do the calculation in `generateId()`.

However:
a) `db-migrate` is clever and doesn't import the file unless it needs it.
b) There is a precedent: `database/src/migrations/20200812043136-AddTongaHpuSettingTypeMapOverlay-modifies-data.js`
c) Doing it this way is very clean, and I can't articulate a reason as to why it shouldn't be done.

Therefore go ahead, I just wanted to mention it I suppose.